eZ/Publish/API/Repository/Values/Content/Query.php

```diff
@@ -137,4 +137,54 @@ public function __isset( $property )
 
         return parent::__isset( $property );
     }
+
+    /**
+     * Returns a human readable, query language like version of the query:
+     *
+     * Example:
+     * ```
+     * contentTypeIdentifier = 'article' AND sectionId = 1 SORT BY dateModified DESCENDING
+     * ```
+     *
+     * @return string
+     */
+    public function __toString()
+    {
+        $stringArray = array();
+        if ( $filterString = (string)$this->filter )
```

Comment:
Need to define how we threat filter vs query.
One option would be to always map FullText to query, and the rest to filter.